index.html

```diff
@@ -133,5 +144,47 @@ <h2>About Me</h2>
     </div>
     </section>
     <br>
+    <section class="footer">
+    <h2 class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in <span class="toll">touch.</span></h2>
+    
+    <form  class="player" action="https://formspree.io/f/xayagqrn" method="post">
+    <ul class="name">
+    <li class="form">
+    <label for="fullName">
+    <input type="text" name="full-name" placeholder="  Full name" maxlength="30" id="fullName" required>
+        </label></li>
+        
+    <li class="form">
+    <label for="email">
+        <input type="email" name="email" id="email" placeholder="  Email address" required>
+        </label></li>
+        
+        <li class="textarea">
+        <label for="message">
+        <textarea name="message" id="message" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
+        </label></li>
+    
+    <li class="buttonclub">
+    <button type="submit" class="button_class">Get in touch</button>
```

Comment:
- [ ] Please try to stick to Figma design, now the button on click has different style: 

Yours `hover`  | Figma
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/145395700-7ac2e78e-bbf6-453c-ab8e-a1e8c28c559d.png)  | ![image](https://user-images.githubusercontent.com/77038610/145396312-cc088a82-f0fe-4c2c-a9f4-cf100c76e901.png)

Reply:
I was able to implement the all the button effect as stated above and as seen in the Figma Template, I look forward to your kind review of my project as I am a beginner and I am working tirelessly to achieve my goals in Microverse.